packages/babel-parser/test/fixtures/typescript/types/const-type-parameters-babel-7/input.ts

```diff
@@ -0,0 +1,24 @@
+function a<const T>() {}
+function b<const T extends U>() {}
+function c<T, const U>() {}
+declare function d<const T>();
+<const T>() => {};
+<const T extends U>() => {};
+
+class A<const T> {}
+class B<const T extends U> {}
+class C<T, const U> {}
+class D<in const T> {}
+class E<const in T> {}
```

Comment:
Can you add some test cases for anonymous function/class expression? They should already work.